### DISCUSSION TITLE
RoboFlow model

### DISCUSSION THREAD

**Discussion by volleyfreak** [shukkkur/VolleyVision#11] [Q&A] [answered]
I struggle to find information about your RoboFlow model. Could you provide a link maybe? All links in the Readme dont get me to a model. 
Or maybe you could just mention the architecture.

**Comment by shukkkur** [comment, marked as answer]
@volleyfreak Hey! 
Badges that I added to the README, such as this:
<a href="https://universe.roboflow.com/shukur-sabzaliev-bh7pq/volleyball-tracking/model/"><img src="https://app.roboflow.com/images/try-model-badge.svg"></img></a>
Will take you to the model page which you can use for inference (API). However, I doubt that you will find the architecture of RoboFlow detection models, but try going over their [docs and blogs](https://blog.roboflow.com/new-and-improved-roboflow-train/).
